Lesson1 (Sequence containers)/Funk.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream>
+#include <vector>
+#include <forward_list>
+
+
+std::vector<int> combineVectors(std::vector<int>::const_iterator begin1, std::vector<int>::const_iterator end1,
+                            std::vector<int>::const_iterator begin2,
+                            std::vector<int>::const_iterator end2)
+{
+    std::vector<int> v;
+    v.reserve((std::distance(begin1,end1)+std::distance(begin2,end2)));
+    v.insert(v.end(), begin1, end1);
+    v.insert(v.end(), begin2, end2);
+    return v;
+}
+
+size_t magicNumber(const size_t digit, const size_t num)
+{
+    std::vector<size_t> v;
+    v.reserve(num);
+    for(size_t i = 0; i < num; i++)
```

Comment:
**try to reduce the lines of code.
this tasks can be solved much easier.**

Did you ever thought why this task is about std::list?